meeteval/wer/wer/error_rate.py

```diff
@@ -58,7 +58,7 @@ def __radd__(self, other: 'int') -> 'ErrorRate':
     def __add__(self, other: 'ErrorRate') -> 'ErrorRate':
         """Combines two error rates"""
         if not isinstance(other, ErrorRate):
-            raise NotImplemented
+            raise NotImplementedError()
```

Comment:
Can you change this to `return NotImplemented`?

Reply:
Done